sig-network/annual-report-2024.md

```diff
@@ -4,61 +4,151 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+This year we have several highlights from our various sub-projects, these are
+the major themes and releases:
+
+- [External DNS](https://github.com/kubernetes-sigs/external-dns)
+  - Two new reviewers were added this year as the project continues to grow
+  - Shipped important maintenance release [v0.15.x](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.15.0)
```

Comment:
It would be great to add to external-dns:
```
   - Created [webhook provider](https://kubernetes-sigs.github.io/external-dns/v0.14.2/tutorials/webhook-provider/) to enable providers running out of tree. Our goal is to move all providers out of tree.
```

Reply:
Sure thing!

```suggestion
  - Shipped important maintenance release [v0.15.x](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.15.0)
  - Created [webhook provider](https://kubernetes-sigs.github.io/external-dns/v0.14.2/tutorials/webhook-provider/) to enable providers running out of tree. Our goal is to move all providers out of tree.
```